### PR TITLE
Update Slint license to v1.2

### PR DESCRIPTION
**Pull request by aurindam**
* Merge the Ambassador and Commercial licenses in one agreement.
* Update the terms to offer the per-Seat (Flex) and per-Product (Buyout) options

### CODE REVIEW

**Review by ogoffart — COMMENTED** [review]
I like that the license is in the repository in markdown format.
(I made a few inline comments regarding just the formatting of markdown)

The thing is that unlike other typical other licenses, this is formulated as a contract to be signed rather than a plain license text. I wonder if that's how to do it.

File: LICENSES/LicenseRef-Slint-commercial.md [hunk 1]
```diff
@@ -1,6 +1,393 @@
...
+
+Obligations:
+   1. Customer grants SixtyFPS the right to use Customer's name and logo in marketing material,
+   2. Customer will include reasonably visible notices in the documentation, marketing materials and social 
+      media channels related to Customer's Products that such were made with the Licensed Software,
+   3. Customer agrees to provide a written testimonial statement about the use of the Licensed Software.
+
+
+**Customer has been granted the right to use the Licensed Software [on payment of applicable Fees / under the Ambassador Program].** (cross out the incorrect option) 
+The permitted number of User Seats and the distribution of Customer's Products is specified below
```
Comment (hunger) [inline LICENSES/LicenseRef-Slint-commercial.md:388]: Should this be replaced with some generic blob about seat/product based licenses?

This seems specific to one customer, and does into a generic statement IMHO,

File: LICENSES/LicenseRef-Slint-commercial.md [hunk 2]
```diff
@@ -1,6 +1,393 @@
...
+
+| Item                                                       |             | Quantity                    |
+|------------------------------------------------------------|-------------|-----------------------------|
+| User Seats                                                 |             | *#UserSeats*            |
+| Distribution of Customer's Products                        |             | *#ProductDistributions*            |
```
Comment (hunger) [inline LICENSES/LicenseRef-Slint-commercial.md:393]: Why did you do 3 columns in this table?

File: LICENSES/LicenseRef-Slint-commercial.md [hunk 3]
```diff
@@ -1,6 +1,393 @@
-## The Slint Commercial License
+# **Slint**
 
-Slint is offered under a commercial license. Please contact info@slint-ui.com if you're interested.
+# **Software License Agreement**
```
[the remark this answers is not in the document]
Comment (aurindam) [reply LICENSES/LicenseRef-Slint-commercial.md:3]: Ok. I will change it to Slint Software License Agreement (dropping the word Commercial as that is implied)

File: LICENSES/LicenseRef-Slint-commercial.md (hunk 1 above)
Comment (aurindam) [reply LICENSES/LicenseRef-Slint-commercial.md:388]: The original agreement was in a format that required signatures from both parties. As @ogoffart also mentioned, maybe we should edit the agreement so that it does not require signatures anymore. One way that I can think of is using a license certificate that is issued to each customer and which mentions the scope of the license grant (whether Ambassador or User Seats (Flex) or Buyout)

File: LICENSES/LicenseRef-Slint-commercial.md (hunk 2 above)
Comment (aurindam) [reply LICENSES/LicenseRef-Slint-commercial.md]: Removed

**Comment by aurindam** [comment]
I have uploaded a modified version of the agreement

**Review by tronical — APPROVED** [review]

**Comment by tronical** [comment]
I don’t see how this relates to the CLA. Can you elaborate?

**Comment by hunger** [comment]
I just reread the CLA: You are right, we do not mention the specific terms of the commercial license there, so I (IANAL, yada, yada) do not see the need to invalidate it.

So let me rephrase: Will this change trigger the CLA bot to invalidate all CLAs and users to click through it again?

**Comment by tronical** [comment]
> So let me rephrase: Will this change trigger the CLA bot to invalidate all CLAs and users to click through it again?

 No, this change - as far as I know - is just adding a file to the repo, that’s it. The CLA is handled by CLA assistant, which in turn uses a gist to retrieve the text of the CLA.

I don’t see any connection, neither of technical nature (due to the way the CLA check is implemented) nor legal nature (see especially section 2.3 of the CLA). However I might be missing something.

**Comment by aurindam** [comment]
Will try a new pull request,
